docs/formal-verification.md

```diff
@@ -0,0 +1,186 @@
+# Formal Verification in Firecracker - Public Documentation
```

Comment:
```suggestion
# Formal Verification in Firecracker
```

Reply:
haha, we noticed this one at the same time xP
fixed :)